### ISSUE TITLE
re-run cmake or configure script in function cvShowImage

### ISSUE THREAD

**Issue by siddas27** [opencv/opencv#9391]
##### System information (version)
- OpenCV => 3.1
- Operating System / Platform => Ubuntu 64 Bit
- Compiler => gcc

##### Detailed description

/root/mc-x64-2.7/conda-bld/opencv-3_1482256854890/work/opencv-3.1.0/modules/highgui/src/window.cpp:545: error: (-2) The function is not implemented. Rebuild the library with Windows, GTK+ 2.x or Carbon support. If you are on Ubuntu or Debian, install libgtk2.0-dev and pkg-config, then re-run cmake or configure script in function cvShowImage

##### Steps to reproduce

```
General configuration for OpenCV 3.1.0 =====================================
  Version control:               unknown

  Platform:
    Host:                        Linux 2.6.18-412.el5 x86_64
    CMake:                       3.6.3
    CMake generator:             Unix Makefiles
    CMake build tool:            /usr/bin/gmake
    Configuration:               Release

  C/C++:
    Built as dynamic libs?:      YES
    C++ Compiler:                /usr/bin/g++  (ver 4.4.7)
    C++ flags (Release):         -I/home/siddharth/anaconda3/include    -fsigned-char -W -Wall -Werror=return-type -Werror=non-virtual-dtor -Werror=address -Werror=sequence-point -Wformat -Werror=format-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -msse -msse2 -mno-avx -msse3 -mno-ssse3 -mno-sse4.1 -mno-sse4.2 -ffunction-sections -fvisibility=hidden -fvisibility-inlines-hidden -fopenmp -O3 -DNDEBUG  -DNDEBUG
    C++ flags (Debug):           -I/home/siddharth/anaconda3/include    -fsigned-char -W -Wall -Werror=return-type -Werror=non-virtual-dtor -Werror=address -Werror=sequence-point -Wformat -Werror=format-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -msse -msse2 -mno-avx -msse3 -mno-ssse3 -mno-sse4.1 -mno-sse4.2 -ffunction-sections -fvisibility=hidden -fvisibility-inlines-hidden -fopenmp -g  -O0 -DDEBUG -D_DEBUG
    C Compiler:                  /usr/bin/cc
    C flags (Release):           -I/home/siddharth/anaconda3/include    -fsigned-char -W -Wall -Werror=return-type -Werror=non-virtual-dtor -Werror=address -Werror=sequence-point -Wformat -Werror=format-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -msse -msse2 -mno-avx -msse3 -mno-ssse3 -mno-sse4.1 -mno-sse4.2 -ffunction-sections -fvisibility=hidden -fopenmp -O3 -DNDEBUG  -DNDEBUG
    C flags (Debug):             -I/home/siddharth/anaconda3/include    -fsigned-char -W -Wall -Werror=return-type -Werror=non-virtual-dtor -Werror=address -Werror=sequence-point -Wformat -Werror=format-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -msse -msse2 -mno-avx -msse3 -mno-ssse3 -mno-sse4.1 -mno-sse4.2 -ffunction-sections -fvisibility=hidden -fopenmp -g  -O0 -DDEBUG -D_DEBUG
    Linker flags (Release):
    Linker flags (Debug):
    Precompiled headers:         YES
    Extra dependencies:          /home/siddharth/anaconda3/lib/libjpeg.so /home/siddharth/anaconda3/lib/libpng.so /home/siddharth/anaconda3/lib/libz.so /home/siddharth/anaconda3/lib/libtiff.so /home/siddharth/anaconda3/lib/libhdf5.so /usr/lib64/librt.so /usr/lib64/libpthread.so /lib64/libz.so /usr/lib64/libdl.so /usr/lib64/libm.so dl m pthread rt
    3rdparty dependencies:       libwebp libjasper IlmImf libprotobuf

  OpenCV modules:
    To be built:                 core flann hdf imgproc ml photo reg surface_matching video dnn fuzzy imgcodecs shape videoio highgui objdetect plot superres xobjdetect xphoto bgsegm bioinspired dpm face features2d line_descriptor saliency text calib3d ccalib datasets java rgbd stereo structured_light tracking videostab xfeatures2d ximgproc aruco optflow stitching python2
    Disabled:                    world contrib_world
    Disabled by dependency:      -
    Unavailable:                 cudaarithm cudabgsegm cudacodec cudafeatures2d cudafilters cudaimgproc cudalegacy cudaobjdetect cudaoptflow cudastereo cudawarping cudev python3 ts viz cvv matlab sfm

  GUI: 
    QT:                          NO
    GTK+:                        NO
    GThread :                    NO
    GtkGlExt:                    NO
    OpenGL support:              NO
    VTK support:                 NO

  Media I/O: 
    ZLib:                        /home/siddharth/anaconda3/lib/libz.so (ver 1.2.8)
    JPEG:                        /home/siddharth/anaconda3/lib/libjpeg.so (ver 80)
    WEBP:                        build (ver 0.3.1)
    PNG:                         /home/siddharth/anaconda3/lib/libpng.so (ver 1.2.10)
    TIFF:                        /home/siddharth/anaconda3/lib/libtiff.so (ver 42 - 4.0.6)
    JPEG 2000:                   build (ver 1.900.1)
    OpenEXR:                     build (ver 1.7.1)
    GDAL:                        NO

  Video I/O:
    DC1394 1.x:                  NO
    DC1394 2.x:                  NO
    FFMPEG:                      NO
      codec:                     NO
      format:                    NO
      util:                      NO
      swscale:                   NO
      resample:                  NO
      gentoo-style:              NO
    GStreamer:                   NO
    OpenNI:                      NO
    OpenNI PrimeSensor Modules:  NO
    OpenNI2:                     NO
    PvAPI:                       NO
    GigEVisionSDK:               NO
    UniCap:                      NO
    UniCap ucil:                 NO
    V4L/V4L2:                    NO/NO
    XIMEA:                       NO
    Xine:                        NO
    gPhoto2:                     NO

  Parallel framework:            OpenMP

  Other third-party libraries:
    Use IPP:                     9.0.1 [9.0.1]
         at:                     /root/mc-x64-2.7/conda-bld/opencv-3_1482256854890/work/opencv-3.1.0/3rdparty/ippicv/unpack/ippicv_lnx
    Use IPP Async:               NO
    Use VA:                      NO
    Use Intel VA-API/OpenCL:     NO
    Use Eigen:                   YES (ver 3.2.8)
    Use Cuda:                    NO
    Use OpenCL:                  NO
    Use custom HAL:              NO

  Python 2:
    Interpreter:                 /home/siddharth/anaconda3/bin/python (ver 2.7.13)
    Libraries:                   /home/siddharth/anaconda3/lib/libpython2.7.so (ver 2.7.13)
    numpy:                       /home/siddharth/anaconda3/lib/python2.7/site-packages/numpy/core/include (ver 1.11.2)
    packages path:               /home/siddharth/anaconda3/lib/python2.7/site-packages

  Python 3:
    Interpreter:                 NO

  Python (for build):            /home/siddharth/anaconda3/bin/python

  Java:
    ant:                         /usr/bin/ant (ver 1.6.5)
    JNI:                         /usr/lib/jvm/java/include /usr/lib/jvm/java/include/linux /usr/lib/jvm/java/include
    Java wrappers:               YES
    Java tests:                  NO

  Matlab:                        NO

  Tests and samples:
    Tests:                       NO
    Performance tests:           NO
    C/C++ Examples:              NO

  Install path:                  /home/siddharth/anaconda3

  cvconfig.h is in:              /root/mc-x64-2.7/conda-bld/opencv-3_1482256854890/work/opencv-3.1.0/build
-----------------------------------------------------------------

    ```

**Comment by alalek** [comment]
> install libgtk2.0-dev and pkg-config

How is the progress?

> /root/mc-x64-2.7/conda-bld

It is better to ask conda package maintainers if they control binaries build process.

**Comment by siddas27** [comment]
I have installed libgtk2.0-dev and pkg-config. what's next?

**Comment by alalek** [comment]
> then re-run cmake

From the original message (but it assumes normal OpenCV build without using of "Conda").

**Comment by siddas27** [comment]
I am a bit confused this shows OpenCV version 3.1.0, but the following
```bash 
pkg-config --modversion opencv
```
shows version
```bash
3.2.0
```
Does that mean I have two different versions installed?

**Comment by alalek** [comment]
The latest released OpenCV version is 3.3.0.
We have no chance to guess how your did receive your OpenCV version and build them. So we can't help a lot here.
Perhaps you should follow to some [OpenCV](http://docs.opencv.org/master/df/d65/tutorial_table_of_content_introduction.html) installation tutorials. There are some good [tutorials](http://www.pyimagesearch.com/opencv-tutorials-resources-guides/) for OpenCV+Python programming.

I'm closing this issue because this place is for issues, bugs and feature requests that needs fix.
Usage questions should go to Users OpenCV Q/A forum: http://answers.opencv.org
